mathics/doc/latex_doc.py

```diff
@@ -1048,6 +1017,6 @@ class LaTeXDocText(DocText):
     Class to hold some (non-test) LaTeX text.
     """
 
-    def latex(self, doc_data) -> str:
+    def latex(self) -> str:
```

Comment:
the parameter doc_data is necessary here because LaTeXDocumentationEntry.latex expects that it accepts the parameter. Otherwise, we need to change the logic there.